app/src/main/java/rick_and_morty/data/event/UIState.kt

```diff
@@ -0,0 +1,12 @@
+package rick_and_morty.data.event
+
+import rick_and_morty.data.model.CharacterResultsDto
+
+data class UIState (
```

Comment:
rename to `CharactersUiState` and move near `CharactersViewModel`. `isLoading = true` by default so it's valid at creation.

Reply:
Done